packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -753,9 +755,15 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   private _onRenderContainer(props: IComboBoxProps): JSX.Element {
     let {
       onRenderList = this._onRenderList,
-      calloutProps
+      calloutProps,
+      comboBoxOptionStyles: customStylesForAllOptions,
+      styles: customStylesForCurrentOption
     } = props;
 
+    let id = this._id;
+    let setWidth = this._comboBoxWrapper.clientWidth - 2;
+    let otherStyles = getOptionStyles(this.props.theme!, customStylesForAllOptions, customStylesForCurrentOption);
```

Comment:
Why isn't the width being passed to getOptionStyles? If you do that then it will get merged and you won't have to set the width on the style attribute below

Reply:
Oh!  I didn't realize it could work like that.  I will update.